lms/templates/navigation.html

```diff
@@ -49,13 +49,21 @@
 </%block>
   <header id="global-navigation" class="header-global ${"slim" if course else ""}" >
     <nav class="wrapper-header" aria-label="${_('Global')}">
-    <h1 class="logo">
-      <a href="${marketing_link('ROOT')}">
-        <%block name="navigation_logo">
-            <img src="${static.url(branding_api.get_logo_url())}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
-        </%block>
-      </a>
-    </h1>
+      <h1 class="logo">
+
+      <%block name="navigation_logo">
+        % if user.is_authenticated():
+          <a href="${reverse('dashboard')}">
```

Comment:
can you just move this starting anchor tag inside the `if` condition? as we are just changing the logo url.